### PR TITLE
Make the changes

### PR DESCRIPTION
**Pull request by codestreamtest**
This is a pull request to test some things.

### CODE REVIEW

File: foo.js [hunk 1]
```diff
@@ -19,6 +19,13 @@ export interface InsertTextCommandArgs {
 	indentAfterInsert?: boolean;
 }
 
+export interface ShowReviewDiffCommandArgs {
+	reviewId: string;
+	checkpoint: CSReviewCheckpoint;
```
Comment (planteater) [inline foo.js:24]: somebody oh

File: foo.js [hunk 2]
```diff
@@ -19,6 +19,13 @@ export interface InsertTextCommandArgs {
...
+	repoId: string;
```
Comment (planteater) [inline foo.js:25]: he needs some milk

File: foo.js [hunk 3]
```diff
@@ -19,6 +19,13 @@ export interface InsertTextCommandArgs {
...
+	path: string;
+}
+
 export interface ApplyMarkerCommandArgs {
 	marker: CSMarkerIdentifier;
```
Comment (planteater) [inline foo.js:30]: one more

File: foo.js [hunk 4]
```diff
@@ -70,6 +77,10 @@ export interface OpenReviewCommandArgs {
 	sourceUri?: Uri;
 }
 
+export interface OpenStreamFooBarArgs {
```
Comment (planteater) [inline foo.js:80]: another comment

File: foo.js [hunk 5]
```diff
@@ -77,6 +77,10 @@ export interface OpenReviewCommandArgs {
...
+	streamThread: StreamThread;
+}
```
Comment (planteater) [inline foo.js:82]: maor change pls

File: foo.js (hunk 1 above)
Comment (planteater) [inline foo.js:24]: forgot this

File: foo.js [hunk 6]
```diff
@@ -19,6 +19,13 @@ export interface InsertTextCommandArgs {
 	indentAfterInsert?: boolean;
 }
 
+export interface ShowReviewDiffCommandArgs {
+	reviewId: string;
```
Comment (codestreamtest2) [inline foo.js:23]: And now a comment in a review from cstest2

**Comment by planteater** [comment]
this is top level - does it get included in the open review?

**Comment by planteater** [comment]
what does this do? just a comment via ext

**Comment by codestreamtest** [comment]
mainline comment

**Comment by planteater** [comment]
mainline by other

File: foo.js [hunk 7]
```diff
@@ -19,6 +19,13 @@ export interface InsertTextCommandArgs {
 	indentAfterInsert?: boolean;
 }
 
+export interface ShowReviewDiffCommandArgs {
```
Comment (planteater) [inline foo.js:22]: Review comment!

File: foo.js [hunk 8]
```diff
@@ -19,6 +19,13 @@ export interface InsertTextCommandArgs {
...
+	reviewId: string;
+	checkpoint: CSReviewCheckpoint;
+	repoId: string;
+	path: string;
+}
```
Comment (planteater) [inline foo.js:27]: Single comment, not in a review

File: foo.js (hunk 7 above)
Comment (planteater) [inline foo.js:22]: This is a review comment again

**Pull request closed without merging**